test/unit/parser.test.js

```diff
@@ -121,7 +121,7 @@ test('includeFile replaces <include src="doesNotExist.md" optional> with empty <
 
   const expected = [
     '# Index',
-    '<div/>',
```

Comment:
consequence of using `xmlMode` previously; `<div></div>` is rendered to `<div/>` which is invalid